docs/release-notes/3.0.0_Beta.md

```diff
@@ -0,0 +1,263 @@
+
+Use **admin** as the username and your EC2 Instance ID as the password (the default PMM password cannot be used for security reasons).
+
+You can find your Instance ID in the AWS Console. For detailed instructions, see [Install PMM Server on AWS Marketplace](../install-pmm/install-pmm-server/aws/aws.md).
+
+### Official ARM support for PMM Client
+
+PMM 3 Beta now officially supports ARM architecture, upgrading from its experimental status in PMM 2.43. This means you can reliably monitor databases on ARM platforms, taking advantage of their cost-effective infrastructure and energy efficiency in data centers and cloud environments.
+
+Installation follows the standard PMM Client process, with no special requirements for ARM systems.  
+
+Test this feature and share your experience on the [PMM forum](https://forums.percona.com/c/percona-monitoring-and-management-pmm/percona-monitoring-and-management-pmm-v2/31) to help accelerate its path to GA.
+
+### Improved UX with Grafana's latest release
+
+PMM now integrates Grafana 11.1.8, which delivers the following important enhancements alongside all the advancements introduced since the previous Grafana 9.2.20 integration in PMM2.
+
+For the full list of  Grafana changes included with this update, see [Grafana’s 11.1.8 changelog](https://community.grafana.com/t/changelog-updates-in-grafana-11-1-8/134843) and [Grafana release highlights](https://grafana.com/docs/grafana/latest/whatsnew/).
+
+#### Improved navigation
+
+PMM now includes a revamped header with search, breadcrumbs, and a reorganized menu that groups related tools together, making it easier to navigate PMM's features and find what you need:
+
+![New Dashboard page](../_images/Navigation_Dashboard.png)
+
+#### Improved Alerting workflow
+
+Leveraging the new Grafana user interface updates, we've taken the opportunity to refine the workflow for creating alerts from the **Alert Rules** and **Alert Rule Templates** pages.
+
+You'll notice separate, more visible options for creating different types of alert rules, cutting down on unnecessary steps and making it easier to manage various alert rules, templates, and configurations:
+
+![New Alerts rule page](../_images/Grafana_Alerting.png)
+
+#### Simplified administration settings with dedicated menus
+
+Administration settings are now easier to manage with the new **Administration** menu. This menu brings together all Grafana-related configurations and account management options that were previously scattered across the general **Configuration** section. This change helps administrators locate and manage Grafana-specific settings more efficiently.
+
+Additionally, all PMM settings and inventory options are now grouped under a new **PMM Configuration** menu. This centralizes access to all PMM-related configurations, making it simpler for users to handle their PMM setup.
```

Comment:
The change from Configuration -> Administration for the Grafana related configuration comes from upstream (from Grafana itself).

As for the "PMM Configuration", I don't personally have any preference there. But it wasn't just renaming Settings -> Configuration, as it now contains all PMM related things (add service, inventory, settings, new updates link).

Reply:
@shoffman-percona i get that the transition looks weird but the daily UX is safe here. If we were to make it:

- PMM Administration
- Administration

I believe it will confuse users on a very subliminal level or lead to misclicks.

User needs the distinction between the two items and we need to avoid clashing with Settings child:

```
├── PMM Configuration
│   ├── + Add service
│   ├── PMM Inventory
│   └── Settings
└── Administration
```

So Configuration for the configuration of PMM, the tool, and Administration for the administration of the accesses